pages/api/lnurlp/[username]/pay.js

```diff
@@ -4,17 +4,21 @@ import { lnurlPayDescriptionHashForUser, lnurlPayMetadataString, lnurlPayDescrip
 import { schnorr } from '@noble/curves/secp256k1'
 import { createHash } from 'crypto'
 import { LNURLP_COMMENT_MAX_LENGTH, MAX_INVOICE_DESCRIPTION_LENGTH } from '@/lib/constants'
-import { toPositiveBigInt } from '@/lib/format'
+import { formatMsats, toPositiveBigInt } from '@/lib/format'
 import assertGofacYourself from '@/api/resolvers/ofac'
 import performPaidAction from '@/api/paidAction'
 import { validateSchema, lud18PayerDataSchema } from '@/lib/validate'
+import { walletLogger } from '@/api/resolvers/wallet'
 
 export default async ({ query: { username, amount, nostr, comment, payerdata: payerData }, headers }, res) => {
   const user = await models.user.findUnique({ where: { name: username } })
   if (!user) {
     return res.status(400).json({ status: 'ERROR', reason: `user @${username} does not exist` })
   }
 
+  const logger = walletLogger({ models, me: user })
+  logger.info('LNURL-pay callback', { amount: formatMsats(amount), nostr, comment })
```

Comment:
"LNURL-pay" might still be confusing but it's better than "lnurlp" imo.

Most stackers are probably the most familiar with the concept of a lightning address, but not sure how to mention that here.